packages/salesforcedx-vscode-core/package.json

```diff
@@ -187,15 +187,7 @@
         },
         {
           "command": "sfdx.force.diff",
-          "when": "sfdx:project_opened && resourceFilename =~ /.*(app|cmp|auradoc|evt|intf|css|design|html|js|svg|permissionset-meta.xml|layout-meta.xml|labels-meta.xml)$/"
-        },
-        {
-          "command": "sfdx.force.diff",
-          "when": "sfdx:project_opened && resourceLangId == apex"
-        },
-        {
-          "command": "sfdx.force.diff",
-          "when": "sfdx:project_opened && resourceLangId == visualforce"
+          "when": "sfdx:project_opened && resourceLangId != 'forcesourcemanifest'"
```

Comment:
Why are we making this change? Just a cleanup or is there any benefit to this?

Reply:
MetadataCacheService allows us to diff more file types, example: `js-meta.xml`

Reply:
This is going to cause the `Diff` command to be exposed for across most contexts ie. a user's `.forceignore`, `.gitignore`, the `manifest` folder etc. Is there anyway to keep this more selective, ie. to just files/directories in the project's source folders?

Reply:
not really, the source folders can be arbitrary. Also we are keeping this consistent with sdr commands. (Deploy/Retreive commands are also available for the aforementioned files)